processor/batchprocessor/README.md

```diff
@@ -15,8 +15,8 @@ any data drops such as sampling.
 Please refer to [config.go](./config.go) for the config spec.
 
 The following configuration options can be modified:
-- `send_batch_size` (default = 8192): Number of spans or metrics after which a
-batch will be sent regardless of the timeout.
+- `send_batch_size` (default = 8192): Number of spans or data point in metrics
+after which a batch will be sent regardless of the timeout.
```

Comment:
Do we care that `batch` is one of the most used processor? Maybe have an option "split_by_metrics_dp" bool that defaults to false and if enable the limit is used for DP?

Reply:
That was my original proposition in #2754, but in the [notes](https://github.com/open-telemetry/opentelemetry-collector/issues/2754#issuecomment-805983282) from the meeting @dashpole says we should change the behaviour altogether. 
Should I get back to introducing the flag?

Reply:
Not sure, @dashpole @tigrannajaryan @jrcamp what do you think?

Also `spans, metric data points, or log records... `

Reply:
Based on what I remember from the discussion, we agreed that it wasn't very useful to batch by number of metrics, as that can give a varying amount of data in each batch.  We couldn't think of a reason why someone would want to batch by number of metrics over number of data points.

IMO if we think there _is_ a use-case for batching by metrics, I would still expect the default to be to batch by the number of data points.

Reply:
I raised this at the sig meeting today, and we were still in agreement that we should change this behavior to batch by number of data points instead of by number of metrics.

Reply:
I agree, for metrics data points makes the most sense as the unit batch operates on.